rllib/core/models/base.py

```diff
@@ -329,8 +327,8 @@ def _forward(self, input_dict: NestedDict, **kwargs) -> NestedDict:
             **kwargs: Forward compatibility kwargs.
 
         Returns:
-            NestedDict: The output tensors. Must contain at a minimum the keys
-                ENCODER_OUT and STATE_OUT (which might be None for stateless encoders).
+            The output tensors. Must contain at a minimum the keys ENCODER_OUT and
```

Comment:
While we are at this, can we also remove the hard-requirement of providing a STATE (and SEQ_LENS?) key in the output, even if the encoder has no internal state?

The design should be that the abstract encoder class just represents a barebones encoder (taking obs), and not an already somewhat specialized (must include STATE), but not yet fully supporting (STATE may be None) abstraction as it is right now.